modules/Hoot/ui/sidebar/advancedOpts.js

```diff
@@ -93,8 +117,80 @@ export default class AdvancedOpts {
                             showingOpts.push(a.name);
                         }
                     } );
+                d3.select('#saveFav').classed('hidden', true);
+                d3.select('#updateFav').classed('hidden', true);
                 this.createGroups(this.advancedOptions, showingOpts);
             });
+
+            favoritesBar
+                .append( 'div' )
+                .classed( 'fav-button-placement', true )
+                .append( 'button' )
+                .classed( 'advanced-opts-reset button secondary strong hidden', true )
+                .attr( 'id', 'saveFav')
+                .text( 'Save Favorites' )
+                .on( 'click', async item => {
+
+                    let currNames = [];
+
+                    this.favoriteOptions.forEach( function(o) { currNames.push(o.name); } );
+
+                    this.saveOpts = new SaveAdvancedOpts( currNames ).render();
+
+                    let favoriteOpts = this.savingFavoriteOpts();
+
+                    this.saveOpts.saveOpts[0].data = favoriteOpts;
+
+                    Hoot.events.once( 'modal-closed', () => delete this.saveOpts );
+
+                    return this;
+                } );
+
+            favoritesBar
+                .append( 'div' )
+                .classed( 'fav-button-placement', true )
+                .append( 'button' )
+                .classed( 'advanced-opts-reset button secondary strong hidden', true )
+                .attr( 'id', 'updateFav')
+                .text( 'Update Favorites' )
+                .on( 'click', function() {
+
+                    let activeFavorite = d3.select( '#conflateType' ).property( 'value' );
+
+                    let toUpdate = instance.savingFavoriteOpts( activeFavorite );
+
+                    let updateOpt = {
+                        name: activeFavorite,
+                        members: {
+                            members: toUpdate,
+                            name: activeFavorite,
```

Comment:
is name inside member used?

Reply:
yeah it's used to build the adv opt panel with the proper css